### PR TITLE
Microsoft.DesktopAppInstaller version 1.17.10271.0

### PR DESCRIPTION
**Pull request by denelon**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/56841)

### CODE REVIEW

**Comment by jedieaston** [comment]
image

File: manifests/m/Microsoft/DesktopAppInstaller/1.17.10271.0/Microsoft.DesktopAppInstaller.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+Publisher: Microsoft Corporation
+PublisherUrl: https://www.microsoft.com/
+PublisherSupportUrl: https://aka.ms/winget-feedback
+PrivacyUrl: http://go.microsoft.com/fwlink/?LinkID=521839
+PackageName: App Installer
+PackageUrl: https://www.microsoft.com/en-us/p/app-installer/9nblggh4nns1
+License: http://go.microsoft.com/fwlink/?LinkID=530144
+LicenseUrl: http://go.microsoft.com/fwlink/?LinkID=530144
+Copyright: (c) 2016 Microsoft Corporation
+ShortDescription: ' Microsoft App Installer for Windows makes sideloading Windows apps easy. Windows Package Manager is supported through App Installer starting on Windows 10 1809.'
```
Comment (Trenly) [inline manifests/m/Microsoft/DesktopAppInstaller/1.17.10271.0/Microsoft.DesktopAppInstaller.locale.en-US.yaml:16]:
```suggestion
ShortDescription: 'Microsoft App Installer for Windows makes sideloading Windows apps easy. Windows Package Manager is supported through App Installer starting on Windows 10 1809.'
```

**Comment by denelon** [comment]
It's more of `winget upgrade winget`, but YES 😁

**Comment by denelon** [comment]
I can also theoretically use this as my reference example for channels in a scenario where "App Installer" can't be installed "side-by-side".

**Comment by Trenly** [comment]
Do we need the `InstallerAbortsTerminal` for this?

**Comment by denelon** [comment]
No, we're performing delayed registration for MSIX. I think it's failing because of the way the validation environment is configured for a different version of the Windows Package Manager. It may require manual verification. I'm also holding on the feature flag for "verified developer" to be enabled for the other referenced PR.

**Review by ItzLevvie — APPROVED** [review]

**Pull request closed without merging**